discord/abc.py

```diff
@@ -417,13 +416,10 @@ async def _edit(
             )
         except KeyError:
             pass
-
-        try:
-            if options.pop("require_tag"):
-                options["flags"] = ChannelFlags.require_tag.flag
-        except KeyError:
-            pass
-
+        if options.get("flags") and not isinstance(
+            options["flags"], int
+        ):  # it shouldn't be an int but just in case
+            options["flags"] = options["flags"].value
```

Comment:
`options["flags"]` shouldn't ever be an integer
```suggestion
```

Reply:
Where should it be made into an integer then ? I'm unsure. Because it needs to be sent as an integer.

Reply:
I meant that it can't be an int at this point but yes it should be converted. See other conversions below

Reply:
So I should only remove the `isinstance` check here ?

Reply:
I updated the code with a try-except block to match the other options, haven't tested.

Reply:
I'll test tomorrow asap

Reply:
Update?

Reply:
It does work, I replaced it with `contextlib.suppress` but if you prefer `try:` `except: pass` I can revert the last commit.